modules/es-extensions/store/extensions/app/greg_store/pages/greg-landing.jag

```diff
@@ -22,7 +22,9 @@ require('/modules/store.js').exec(function(ctx) {
 
 	var log = new Log();
 	log.debug("Printing user ############" + stringify(user));
-	var tenantId = user.tenantId;
+    
+    var tenantDetails = tenantApi.createTenantDetails(request, session);
```

Comment:
I have actually formatted that before commit and cannot see this alignment issue on the IDE. Somehow Github shows it wrongly.